CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing to Open Liberty Guides
+
+Anyone can contribute to the Open Liberty guides and we welcome your contributions!
+
+There are multiple ways to contribute: report bugs, fix bugs, contribute code, improve upon documentation, etc. 
+
+## Raising Issues
+
+Please raise any bug reports on the Open Liberty guide's repository: https://github.com/OpenLiberty/[guide-name]/issues, where `[guide-name]` is the repository name of the guide . For new topics, large updates to existing guides, or general suggestions and ideas, report your issue on the [Open Liberty common guides repository](https://github.com/OpenLiberty/guides-common/issues). Be sure to search the list to see if your issue has already been raised.
+
+A good bug report is one that makes it easy for everyone to understand what you were trying to do and what went wrong. Provide as much context as possible so we can try to recreate the issue.
+
+## Contributions 
+
+Contributing to the Open Liberty guides is simple. All you need to do is create your own branch to include your contribution(s). If you want to make updates to existing guides, create a new branch and open a pull request to the `qa` (quality assurance) branch.
```

Comment:
- [ ] "...to existing guides, create a new branch..." it should not be possible to have a branch to update guides.

Reply:
Isn't the contributor creating a new branch for their changes?

Reply:
will be addressed by gkwan-ibm-patch-1 branch